### ISSUE TITLE
DOTCOM-25385 - Accessibility: iframe elements are missing a title attribute

### ISSUE THREAD

**Issue by rofe** [adobe/theblog#331]
From: https://jira.corp.adobe.com/browse/DOTCOM-25385

Example URLs:

https://blog.adobe.com/en/2020/07/09/prototypes-enhance-remote-design-dev-collaboration.html

https://blog.adobe.com/en/2020/04/14/using-creativity-to-honorheroes-doing-extraordinary-things-during-covid-19.html

https://blog.adobe.com/en/2020/07/08/adobe-document-cloud-debuts-education-resource-hub.html

https://blog.adobe.com/en/2020/07/08/adobe-document-cloud-debuts-education-resource-hub.html

**Reproduction Steps:**

Open the above listed URLs and look for an iframe. Looks like they're all from embedded videos. These iframe elements are missing a title attribute.

**Expected behavior:**

Screen reader users rely on a frame title to describe the contents of the frame. Navigating through frame and iframe elements quickly becomes difficult and confusing for users of this technology if the markup does not contain a title attribute.

Screen reader users have the option to pull up a list of titles for all frames on a page. Adding descriptive, unique titles allows users to quickly find the frame they need. If no titles are present, navigating through frames can quickly become difficult and confusing. If no title is listed, screen readers will instead give information like “frame,” “JavaScript,” the filename, or the URL. In most cases, this information won’t be very helpful.

**WCAG Success Criteria**

[Name Role Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Comment by rofe** [comment]
@trieloff I assume we inherit the iframe element from `iframely` - should we log an issue in https://github.com/itteco/iframely?

**Comment by trieloff** [comment]
We could go through their support, but an easier way to fix this would be to rewrite the HTML in helix-embed.

Iframely is getting a large amount of the HTML directly from the provider, which might not care about the accessibility.

**Comment by rofe** [comment]
Fixed via https://github.com/adobe/helix-embed/issues/286
